modules/reactor/src/meshgenerators/PolygonMeshGeneratorBase.C

```diff
@@ -338,6 +419,7 @@ void
 PolygonMeshGeneratorBase::backgroundNodes(ReplicatedMesh & mesh,
                                           const unsigned int num_sectors_per_side,
                                           const unsigned int background_intervals,
+                                          const std::vector<Real> biased_terms,
```

Comment:
biased terms are no longer 1 for no-bias right?
They are 1/number of intervals right?

Reply:
Yes.